<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MKoligo</class>
 <widget class="QMainWindow" name="MKoligo">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1070</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1070</width>
    <height>673</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CSPSynth</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Desktop/ico.ico</normaloff>../Desktop/ico.ico</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>PCR-Assembly</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="CSVgroupBox">
            <property name="title">
             <string>PCR-assembly</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="pushButton_6">
                   <property name="text">
                    <string>Load File</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTextEdit" name="textEdit">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="horizontalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="plainTextEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Analise sequence</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>minimum Tm:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QDoubleSpinBox" name="doubleSpinBox">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <double>99.900000000000006</double>
                   </property>
                   <property name="value">
                    <double>60.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="maximumSize">
                    <size>
                     <width>25</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;O&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>maximum length of primers:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QSpinBox" name="spinBox">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>300</number>
                   </property>
                   <property name="value">
                    <number>60</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="maximumSize">
                    <size>
                     <width>25</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;nt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>minimum length of primers:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QSpinBox" name="spinBox_2">
                   <property name="minimum">
                    <number>3</number>
                   </property>
                   <property name="maximum">
                    <number>200</number>
                   </property>
                   <property name="value">
                    <number>15</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="maximumSize">
                    <size>
                     <width>25</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;nt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>RUN</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Файл настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
